### DISCUSSION TITLE
Recieving a notification when getting a call from an associated external webapp

### DISCUSSION THREAD

**Discussion by AlbertoPerez8** [Azure/communication-ui-library#2738] [Q&A]
I am working on building a communications app built on the Teams Toolkit that could be able to communicate with external participants through a dedicated website. One of the planned features for implementation is to be able to create and answer video conference calls. Are there any components or features available that can help signal either endpoint party member that a call has been initiated so as to be able to join, or is it not possible to do since it is two independent applications?

**Comment by alkwa-msft** [comment]
Not at this time. The UI Library builds directly on top of the calling and chat sdks so if this functionality is not available through those underlying sdks we are probably not able to support them in the UI Library at this time. 

We are always looking to improve so if you are aware of how this signaling can be occurring then we can put something together.

I believe the crux here is that Teams clients can't call out to ACS clients. If they could then its just a matter of registering for listening for calling events and then to answer the call. I would suggest playing around with the JS Calling SDK to see if the functionality you are looking for matches what you need?
https://www.npmjs.com/package/@azure/communication-calling
